.gitignore

```diff
@@ -0,0 +1,15 @@
+# RUST
```

Comment:
1. Why would we add a gitignore for Rust without using Rust? I would wait for the first Rust code first.
2. I (and the guys from your Medium article) have good experience with this [gitignore generator](https://www.toptal.com/developers/gitignore) and would recommend to use it. I did this already on #46, we could cherry pick that commit here.

Reply:
> 1. Why would we add a gitignore for Rust without using Rust? I would wait for the first Rust code first.
> 2. I (and the guys from your Medium article) have good experience with this [gitignore generator](https://www.toptal.com/developers/gitignore) and would recommend to use it. I did this already on #46, we could cherry pick that commit here.
> 

I agree, I just wanted sth rather short and code related in order to demonstrate the path layout in the gitignore

Reply:
We could also use 2 gitignores to break things up; 1 in /src and 1 in /hw